<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LocalSettings</class>
 <widget class="QWidget" name="LocalSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="localSettingsLabel">
     <property name="text">
      <string>These settings apply only to this widget.</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="topGridLayout" columnstretch="1,1">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="deviceHorizontalLayout">
       <item>
        <widget class="QLabel" name="selectedDeviceLabel">
         <property name="text">
          <string>Selected &amp;device:</string>
         </property>
         <property name="buddy">
          <cstring>selectedDeviceComboBox</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="selectedDeviceComboBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'DejaVu Sans'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Choose a specific network device to monitor or choose &quot;any&quot;&lt;span style=&quot; font-style:italic;&quot;&gt; &lt;/span&gt;to see traffic across all supported devices.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="whatsThis">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="showFilterSortCheckBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Show or hide the quick search and freeze sort controls.</string>
       </property>
       <property name="text">
        <string>Show search and sort controls</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="KButtonGroup" name="groupByButtonGroup">
       <property name="toolTip">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'DejaVu Sans'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Choose the grouping criteria for rows. &lt;span style=&quot; font-weight:600;&quot;&gt;Connection&lt;/span&gt; shows the most detail. &lt;span style=&quot; font-weight:600;&quot;&gt;Host pair and process&lt;/span&gt; combines connections that differ only by port or protocol. &lt;span style=&quot; font-weight:600;&quot;&gt;Host pair and program&lt;/span&gt; further combines connections that differ only by user or process ID.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="title">
        <string>Show one row per:</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="connectionRadioButton">
          <property name="text">
           <string>&amp;Connection</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="hostPairProcessRadioButton">
          <property name="text">
           <string>&amp;Host pair and process</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="hostPairProgramRadioButton">
          <property name="text">
           <string>Host pair and &amp;program</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QGroupBox" name="subdomainLevelsGroupBox">
       <property name="toolTip">
        <string>Choose how many levels of subdomain details to show when host name lookups are enabled.</string>
       </property>
       <property name="title">
        <string>Show subdomain levels:</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QRadioButton" name="subdomainLevels1Button">
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QRadioButton" name="subdomainLevels3Button">
          <property name="text">
           <string>3</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QRadioButton" name="subdomainLevelsAllButton">
          <property name="text">
           <string>All</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="subdomainLevels2Button">
          <property name="text">
           <string>2</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="columnsGroupBox">
     <property name="title">
      <string>Columns</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="KActionSelector" name="columnsSelector">
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Select the set of columns to be made visible and the order you would like them to appear.</string>
        </property>
        <property name="availableInsertionPolicy">
         <enum>KActionSelector::Sorted</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KButtonGroup</class>
   <extends>QGroupBox</extends>
   <header>kbuttongroup.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KActionSelector</class>
   <extends>QWidget</extends>
   <header>kactionselector.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
